src/CMakeLists.txt

```diff
@@ -37,11 +37,20 @@ endif()
 # Expose public API
 target_include_directories(benchmark PUBLIC ${PROJECT_SOURCE_DIR}/include)
 
+# Pick installation dir depending on bitness if not specified
+if(NOT DEFINED CMAKE_INSTALL_LIBDIR)
```

Comment:
We should get the default value of `CMAKE_INSTALL_LIBDIR` by including the GNUInstallDirs.cmake module, not by setting it ourselves.

After fixing that this patch LGTM.